runner/test/cases/server/plugins/engine/pageControllers/getPageController.test.ts

```diff
@@ -77,4 +79,74 @@ suite("Engine Page Controllers getPageController", () => {
re","schema":{}},{"type":"TextField","name":"lastName","title":"Surname","options":{"required":true},"schema":{}}],"next":[{"path":"/applicant-four-address"}]},{"path":"/applicant-four-address","section":"applicantFourDetails","components":[{"type":"UkAddressField","name":"address","title":"Address","options":{"required":true},"schema":{}}],"next":[{"path":"/contact-details"}],"title":"Address"},{"path":"/contact-details","section":"applicantDetails","components":[{"type":"TelephoneNumberField","name":"phoneNumber","title":"Phone number","hint":"If you haven\'t got a UK phone number, include country code","options":{"required":true},"schema":{}},{"type":"EmailAddressField","name":"emailAddress","title":"Your email address","options":{"required":true},"schema":{}}],"next":[{"path":"/testconditions"}],"title":"Applicant contact details"},{"path":"/summary","controller":"./pages/summary.js","title":"Summary","components":[],"next":[]},{"path":"/testconditions","title":"TestConditions","components":[{"name":"pmmRYP","options":{"condition":"bDDfgf"},"type":"Para","content":"There Is Someone Called Martin"}],"next":[{"path":"/summary"}]}],"lists":[{"name":"numberOfApplicants","title":"Number of people","type":"number","items":[{"text":"1","value":1,"description":"","condition":""},{"text":"2","value":2,"description":"","condition":""},{"text":"3","value":3,"description":"","condition":""},{"text":"4","value":4,"description":"","condition":""}]}],"sections":[{"name":"checkBeforeYouStart","title":"Check before you start"},{"name":"applicantDetails","title":"Applicant details"},{"name":"applicantOneDetails","title":"Applicant 1"},{"name":"applicantTwoDetails","title":"Applicant 2"},{"name":"applicantThreeDetails","title":"Applicant 3"},{"name":"applicantFourDetails","title":"Applicant 4"}],"phaseBanner":{},"fees":[],"outputs":[{"name":"Ric43H5Ctwl4NBDC9x1_4","title":"email","type":"email","outputConfiguration":{"emailAddress":"jennifermyanh.duong@digital.homeoffice.gov.uk"}}],"declaration":"<p class=\\"govuk-body\\">All the answers you have provided are true to the best of your knowledge.</p>","version":2,"conditions":[{"name":"hasUKPassport","displayName":"hasUKPassport","value":"checkBeforeYouStart.ukPassport==true"},{"name":"doesntHaveUKPassport","displayName":"doesntHaveUKPassport","value":"checkBeforeYouStart.ukPassport==false"},{"name":"moreThanOneApplicant","displayName":"moreThanOneApplicant","value":"applicantDetails.numberOfApplicants > 1"},{"name":"moreThanTwoApplicants","displayName":"moreThanTwoApplicants","value":"applicantDetails.numberOfApplicants > 2"},{"name":"moreThanThreeApplicants","displayName":"moreThanThreeApplicants","value":"applicantDetails.numberOfApplicants > 3"},{"name":"bDDfgf","displayName":"testCondition","value":{"name":"testCondition","conditions":[{"field":{"name":"applicantOneDetails.firstName","type":"TextField","display":"First name"},"operator":"is","value":{"type":"Value","value":"Martin","display":"Martin"}}]}}]}'
```

Comment:
you should have this as a separate JSON file and read it with fs.readFileSync, or you can have a js file with the parsed json 
`export default {...json}`, so you can import it like so `import file from './getPageController.config.js'. 

it'll make it easier to edit the config further down the line

Reply:
Thanks, turns out we wanted to include the form json for the new smoke-tests anyway so we're just importing it into our unit tests now too.